doc/source/reference/arrays.dtypes.rst

```diff
@@ -87,18 +87,18 @@ Sub-arrays always have a C-contiguous memory layout.
 
    >>> dt = np.dtype([('name', np.unicode_, 16), ('grades', np.float64, (2,))])
    >>> dt['name']
-   dtype('|U16')
+   dtype('<U16')
```

Comment:
Curious why the endianess should be specified here, why not just `'U16'`?

Reply:
This is what is returned when defining the dtype with np.unicode_. I can't find anything in the docs or by googling on what is "expected". Only in `numpy/lib/tests/test_io.py:1581` can you find an instance of `|U11` but it returns `<U11` on my machine.